app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/actions.ts

```diff
@@ -66,3 +69,29 @@ export const resetThrottlingRate = AuthenticatedAction(async (session, formId: s
     return { error: "There was an error. Please try again later." } as ServerActionError;
   }
 });
+
+export const updateNotificationsInterval = AuthenticatedAction(
+  async (
+    session,
+    formId: string,
+    notificationsInterval: NotificationsInterval,
+    isPublished: boolean
+  ) => {
+    // The form config could instead be pulled from the DB to check if the form is published if this needs more security
+    if (isPublished) {
+      return;
+    }
+    Promise.all([
+      updateNotificationsSetting(formId, notificationsInterval),
+      // Remove old cache value to allow a new one with the new ttl to be created when the next submission is sent
+      removeMarker(formId),
+    ]).then(() =>
+      logEvent(
```

Comment:
This more descriptive logEvent could be moved to `updateNotificationSettings` where there is a somewhate identical logEvent also occuring.